runtime/src/commonMain/kotlin/pbandk/ser/TimestampSerializer.kt

```diff
@@ -4,13 +4,13 @@ import kotlinx.serialization.*
 import pbandk.Util
 import pbandk.wkt.Timestamp
 
-@Serializer(forClass = Timestamp.JsonMapper::class)
-object TimestampSerializer: KSerializer<Timestamp.JsonMapper> {
+@Serializer(forClass = Timestamp::class)
+object TimestampSerializer: KSerializer<Timestamp> {
```

Comment:
Was there a reason for changing this? From what I remember, this `TimestampSerializer` is only used for JSON serialization. The way we have JSON serialization currently implemented, the actual object that gets serialized is the `Timestamp.JsonMapper` class, not the `Timestamp` class. So I think your change will break JSON serialization of `Timestamp` messages.

Reply:
So since I the `JsonMapper` isn't getting created, when generate json is off, this was causing an error.  I am looking into how to fix. Perhaps keep `JsonMapper`

Reply:
`Timestamp` is bundled as part of the pbandk runtime library. The bundled version should be generated with JSON enabled, so you can depend on the `JsonMapper` being present.